client/app/scripts/components/node-details/node-details-table.js

```diff
@@ -50,6 +51,16 @@ const COLUMN_WIDTHS = {
 };
 
 
+// TODO: Consider introducing the 'ip' dataType for
+// the fields instead of maintaining a list here.
+const IP_FIELDS = ['kubernetes_ip'];
```

Comment:
I found the place(s) where I need to modify it on the backend, but I can't really test it manually right now, as I'd need to run the updated backend code and generate a new report that I'd need to plug into the frontend client that I'm running locally. I'm sure there is a better way to dynamically observe changes on the backend, but I don't know yet how to run the backend service in the development mode - maybe you guys could show me that on Monday?

So, I'll just stash those changes for now and play with some other stuff the rest of the day :)